shm/Game.hpp

```diff
@@ -19,6 +19,7 @@ class Game {
     Map* map_{nullptr};
     std::shared_ptr<Player> player_{nullptr};
     Time* time_{nullptr};
+    Island* currentPosition_{nullptr};
```

Comment:
I've been working on converting raw pointers to smart ones(forgot to push). can this be unique_ptr?

Reply:
done